modules/oadp-release-notes-1-3-4.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes-1-3.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-release-notes-1-3-4_{context}"]
+= {oadp-short} 1.3.4 release notes
+
+The {oadp-first} 1.3.4 release notes list resolved issues and known issues.
+
+[id="resolved-issues-1-3-4_{context}"]
+== Resolved issues
+
+.The backup spec.resourcepolicy.kind parameter is now case insensitive
+
+Previously, the `backup spec.resourcepolicy.kind` parameter was only supported with a lower-level string. With this fix, it is now case insensitive. link:https://issues.redhat.com/browse/OADP-2944[{oadp-short}-2944]
+
+.Use olm.maxOpenShiftVersion to prevent cluster upgrade to OCP 4.16 version
+
+The cluster `operator-lifecycle-manager` operator must not be upgraded between minor {OCP} versions. Using the `olm.maxOpenShiftVersion` parameter prevents upgrading to {OCP} 4.16 version when {oadp-short} 1.3 is installed. 
+To upgrade to {OCP} 4.16 version, upgrade {oadp-short} 1.3 on OCP 4.15 version to {oadp-short} 1.4. link:https://issues.redhat.com/browse/OADP-4803[{oadp-short}-4803]
+
+.BSL and VSL are removed from the cluster
+
+Previously, when any Data Protection Application (DPA) was modified to remove the Backup Storage Locations (BSL) or Volume Snapshot Locations (VSL) from the `backupLocations` or `snapshotLocations` section, BSL or VSL were not removed from the cluster until the DPA was deleted.
+With this update, BSL/VSL are removed from the cluster. link:https://issues.redhat.com/browse/OADP-3050[{oadp-short}-3050]
+
+.DPA reconciles and validates the secret key
+
+Previously, the Data Protection Application (DPA) reconciled successfully on the wrong Volume Snapshot Locations (VSL) secret key name.
+With this update, DPA validates the secret key name before reconciling on any VSL. link:https://issues.redhat.com/browse/OADP-3052[{oadp-short}-3052]
+
+
+.Velero's cloud credential permissions are now restrictive
+
+Previously, Velero's cloud credential permissions were mounted with the 0644 permissions. As a consequence, any one could read the `/credentials/cloud` file apart from the owner and group making it easier to access sensitive information such as storage access keys.
```

Comment:
Just wondering if we should be explicit about `rw-r--r--` or even 
* user: read and write
* group: read
* others: read